### ISSUE TITLE
UI: turn off spell checking on input fields

### ISSUE THREAD

**Issue by Bill-Kunj** [rchain-community/rgov#226]
The red squiggly shows up on most of our input fields because we're not typing natural language.
The input markups should turn this off.

**Comment by David405** [comment]
I was thinking we fixed this already, I can't see the red lines when I edit

**Comment by Bill-Kunj** [comment]
Yes, this is true for the ```<textarea>``` editor element. However, this is not true for the other input elements generated by participate.js.

**Comment by Bill-Kunj** [comment]
@David405 This no longer happens on the official https://rchain-community.github.io/rgov/ link. Can this be closed?
